### PR TITLE
large_semi_axis_of_orbit

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: symplyphysics/laws/astronomy/large_semi_axis_of_orbit.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Let the body move in an elliptical orbit. Then its large semi-axis depends on the mass of the body around which it
+## rotates and the orbital velocity.
+
+## Law is: a = G * M / v^2, where
+## a - large semi-axis of the orbit,
+## G - gravitational constant,
+## M - mass of the body around which the movement takes place,
+## v - orbital velocity.
+
+large_semi_axis = Symbol("large_semi_axis", units.length)
```
Comment (blackyblack) [inline symplyphysics/laws/astronomy/large_semi_axis_of_orbit.py:23]: I suggest to update https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/gravity/orbital_velocity_from_length_of_large_half_axis_and_planet_mass.py to have the same names of variables. Also you can move that law to `astronomy`

File: test/astronomy/large_semi_axis_of_orbit.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors,)
+from symplyphysics.laws.astronomy import large_semi_axis_of_orbit as axis_law
+
+# Description
+## The mass of the sun is 1.989e30 kilograms, and the orbital speed of the Earth is 29.8 kilometers per second.
+## Then the semi-major axis of the orbit is equal to 149.5e9 meters.
+## https://uchet-jkh.ru/i/kak-naiti-bolsuyu-poluos-orbity/
```
Comment (blackyblack) [inline test/astronomy/large_semi_axis_of_orbit.py:9]: This link has this law reversed, yet the same result as yours. How is it possible?

File: test/astronomy/large_semi_axis_of_orbit.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+def test_args_fixture() -> Args:
+    orbital_velocity = Quantity(29.8 * (units.kilometer / units.second))
+    mass = Quantity(1.989e30 * units.kilogram)
+
+    return Args(orbital_velocity=orbital_velocity, mass=mass)
+
+
+def test_basic_large_semi_axis(test_args: Args) -> None:
+    result = axis_law.calculate_large_semi_axis(test_args.orbital_velocity, test_args.mass)
+    assert_equal(result, 149.5e9 * units.meter)
```
Comment (blackyblack) [inline test/astronomy/large_semi_axis_of_orbit.py:24]: I don't really like non standard scientific notation, where coefficient is larger than 10. But for the sake of conforming to the example we can keep it this way.

File: symplyphysics/laws/astronomy/large_semi_axis_of_orbit.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    validate_output,
+    angle_type,
+)
+from sympy.physics.units import gravitational_constant
+
+# Description
+## Let the body move in an elliptical orbit. Then its large semi-axis depends on the mass of the body around which it
+## rotates and the orbital velocity.
+
+## Law is: a = G * M / v^2, where
```
[the remark this answers is not in the document]
Comment (Zulusssss) [reply symplyphysics/laws/astronomy/large_semi_axis_of_orbit.py:17]: > Could you write a proof of give me a link to this law?

https://uchet-jkh.ru/i/kak-naiti-bolsuyu-poluos-orbity/

image

File: test/astronomy/large_semi_axis_of_orbit.py (hunk 2 above)
Comment (blackyblack) [reply test/astronomy/large_semi_axis_of_orbit.py:9]: Someone following the link might be confused. Add some comments about missing `-1` power.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**